system-tests/CUserHeader_test.py

```diff
@@ -0,0 +1,27 @@
+##############################################################################
+# (c) Crown copyright Met Office. All rights reserved.
+# For further details please refer to the file COPYRIGHT
+# which you should have received as part of this distribution
+##############################################################################
+from pathlib import Path
+from common import CompareConsoleWithFile, RunFab, RunExec, RunDump
+
+TEST_PATH = Path('system-tests/CUserHeader')
```

Comment:
I'm a bit concerned that this `TEST_PATH` is appearing in every test package. Is there a way to improve this? Maybe their similarity illusory and they should be defined in each case.

Reply:
I wondered actually if it could be extracted from the routine name?  If we are going to keep a 1:1 correspondence with the test filename and the test directory name?